criteria/inmemory/src/org/immutables/criteria/inmemory/ExpressionInterpreter.java

```diff
@@ -130,7 +130,7 @@ public Object visit(Call call) {
       }
 
       if (op == StringOperators.TO_UPPER_CASE || op == StringOperators.TO_LOWER_CASE) {
-        Preconditions.checkArgument(args.size() == 1, "Size should be 1 but was %d", args.size());
```

Comment:
These are actually errorprone errors. It knows that Preconditions doesn't accept anything other than `%s` as a placeholder.